### DISCUSSION TITLE
alt+space not working

### DISCUSSION THREAD

**Discussion by git-78** [sumatrapdfreader/sumatrapdf#3223] [Q&A] [answered]
Hi,

I'm using a lot in any application the alt+space shortcut to open an app's system menu. It used to work in the past in Sumatrapdf, but not on v3.4.6 64bit (on Win10). Does it work for you? Is there any workaround? Or is it a bug ?

Thanks for your answer, and thanks a lot to the dev for this excellent application

**Comment by GitHubRulesOK** [comment]
it works for me in both current pre-rel version and 3.4.6 x64 (Windows 10 Pro  22H2)
![image](https://user-images.githubusercontent.com/12754913/212556836-21112e23-431a-4456-965e-44d03a090d7b.png)
![image](https://user-images.githubusercontent.com/12754913/212556883-838b9a88-e115-490c-8fa2-515b0381a422.png)

**Comment by git-78** [comment, marked as answer]
I closed Sumatrapdf and reopened it... and it works. Weird but efficient :-)
